.golangci.yaml

```diff
@@ -82,18 +82,19 @@ linters-settings:
   revive:
     enable-all-rules: true
     rules:
-      - name: add-constant
-        disabled: true
-      - name: bare-return
-        disabled: true
-      - name: blank-imports
-        disabled: true
+      # Configuration for enabled rules
       - name: cognitive-complexity
         arguments: [30]
-      - name: confusing-naming
-        disabled: true
       - name: cyclomatic
         arguments: [15]
+      - name: unhandled-error
+        arguments:
+          - "bytes.Buffer.WriteString" # Always returns nil error
+      # TODO: work on these
```

Comment:
Does this have a task already? Just so we don't forget. And surprise me a bit that the linter doesn't complain about todos hehe

Reply:
https://github.com/elastic/cloudbeat/issues/1669 tracks the progress on the rules